package.json

```diff
@@ -7,7 +7,7 @@
     "react-dom": "^15.6.1",
     "react-scripts": "1.0.10"
   },
-  "homepage": "https://bsides.github.io/horizoverlay",
```

Comment:
Could you revert that or maybe erase it altogether? I'm thinking about deploying elsewhere anyway (now.sh seems to be good for it)